src/WebUI/Components/ServicesComponent.tsx

```diff
@@ -24,21 +24,32 @@ const ageKey: string = "age-col";
 
 export interface IServicesComponentProperties extends IVssComponentProperties {
     servicesList: V1ServiceList;
+    typeSelections: string[];
+    nameFilter?: string,
     onItemInvoked?: (item?: any, index?: number, ev?: Event) => void;
 }
 
 export class ServicesComponent extends BaseComponent<IServicesComponentProperties, {}> {
     public render(): React.ReactNode {
-
-        return (
-            <ListComponent
-                className={css("list-content", "depth-16")}
-                items={ServicesComponent._getServiceItems(this.props.servicesList)}
-                columns={ServicesComponent._getColumns()}
-                onRenderItemColumn={ServicesComponent._onRenderItemColumn}
-                onItemInvoked={this._openServiceItem}
-            />
-        );
+        const filteredSvc: V1Service[] = (this.props.servicesList && this.props.servicesList.items || [])
+            .filter((svc) => {
+                return this._filterService(svc);
+            });
+        if (filteredSvc.length > 0) {
+            return (
+                <div>{

```

Comment:
if <div> is needed, then have a className in it, otherwise remove it.

Reply:
Removed